app/models/ore/aggregation.rb

```diff
@@ -53,17 +55,40 @@ def edm_ugc_enum
     delegate :dc_language, :dc_title, to: :edm_aggregatedCHO
     delegate :edm_ugc_enum, to: :class
     delegate :media, to: :edm_isShownBy, allow_nil: true
-    delegate :campaign, :draft?, :published?, :deleted?, :ever_published?, to: :contribution, allow_nil: true
+    delegate :campaign, :ever_published?, to: :contribution, allow_nil: true
 
     validates :edm_ugc, inclusion: { in: edm_ugc_enum }
     validates :edm_provider, :edm_dataProvider, presence: true
-    validates_associated :edm_aggregatedCHO
+    validates_associated :edm_aggregatedCHO, :edm_isShownBy, :edm_hasViews
```

Comment:
edm_isShownBy and edm_hasViews aren't needed here as mongoid will do this by default: https://docs.mongodb.com/mongoid/7.0/tutorials/mongoid-relations/index.html#validations